double-check-tmp.md

```diff
@@ -0,0 +1,28 @@
+- No copyright/license mentioned
+  + http://www.jjj.de/fxt/fxtbook.pdf
+  + http://igm.univ-mlv.fr/~mac/REC/text-algorithms.pdf
+  + http://www.ethoberon.ethz.ch/WirthPubl/CBEAll.pdf
+  + http://lampwww.epfl.ch/~schinz/thesis-final-A4.pdf
+  + http://www.stack.nl/~marcov/compiler.pdf
```

Comment:
This is an interesting one. The author (http://jackcrenshaw.com/)  links to a different website, which is probably the one we should link to: http://compilers.iecc.com/crenshaw/ (it has more version, too.)